src/bugsnag.js

```diff
@@ -504,7 +504,6 @@
   }
 
   // Compare if two objects are equal.
-  // TODO check if this would fail if the properties are traveresed in different orders
   function isEqual(obj1, obj2) {
     serialize(obj1) === serialize(obj2);
```

Comment:
Me thinks there also might be a `return` missing? Unless you're using it for testing right now, in which case I'll get lost and quit bugging you :P